Menu/Menu.js

```diff
@@ -1,10 +1,23 @@
 
-const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+// const toggleMenu = (e) => {
+//   menu.classList.add("menu--open");
+// }
+
+const toggleMenu = (e) => {
+  if (e.target === menuButton) {
+    menu.classList.add("menu--open")
+  } else if (!e.target.closest(".menu")) {
+    menu.classList.remove("menu--open")
+  };
```

Comment:
When the menu is open you should be able to click the hamburger menu to close it. 
`menu.classList.toggle("menu--open");` would have been more appropriate.